includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1466,6 +1493,65 @@ class="jetpack-logo"
 		<?php
 	}
 
+	protected function get_all_activate_errors() {
+		return array(
+			'default' => __( "Sorry! We tried, but we couldn't connect Jetpack just now 😭. Please go to the Plugins tab to connect Jetpack, so that you can finish setting up your store.", 'woocommerce' ),
+			'jetpack_cant_be_installed' => __( "Sorry! We tried, but we couldn't install Jetpack for you 😭. Please go to the Plugins tab to install it, and finish setting up your store.", 'woocommerce' ),
+			'jetpack_cant_be_activated' => __( "Sorry! We tried, but we couldn't activate Jetpack for you 😭. Please go to the Plugins tab to activate it, and finish setting up your store.", 'woocommerce' ),
+			'register_http_request_failed' => __("Sorry! We couldn't contact Jetpack just now 😭. Please make sure that your site is visible over the internet, and that it accepts incoming and outgoing requests via curl. You can also try to connect to Jetpack again, and if you run into any more issues, please contact support.", 'woocommerce'),
+			'siteurl_private_ip_dev' => __( "Whoops! We couldn't connect. Your site is probably on a private network. Jetpack can only connect to public sites. Please make sure your site is visible over the internet, and then try connecting again 🙏." , 'woocommerce'),
```

Comment:
Nitpick: whitespace near parentheses (the line above as well).

Reply:
Fixed & rebased - now correctly spaced in 85dd98da22967655ea244e55ce5d5c32e52f3e11